module/x/gravity/cosmos-originated_test.go

```diff
@@ -191,7 +190,7 @@ func acceptDepositEvent(tv *testingVars, myNonce uint64) {
 	// Check that gravity balance has gone down
 	gravityAddr := tv.input.AccountKeeper.GetModuleAddress(types.ModuleName)
 	assert.Equal(tv.t,
-		sdk.Coins{sdk.NewCoin(tv.denom, sdk.NewIntFromUint64(55).Sub(myErc20.Amount))},
+		sdk.Coins{sdk.NewCoin(tv.denom, sdk.NewIntFromUint64(150).Sub(myErc20.Amount))},
```

Comment:
I don't fully understand this test or why I had to change this, did I break its purpose?